packages/backend-common/src/reading/GithubUrlReader.test.ts

```diff
@@ -69,21 +72,11 @@ const gheProcessor = new GithubUrlReader(
   { treeResponseFactory, credentialsProvider: mockCredentialsProvider },
 );
 
-const tmpDir = os.platform() === 'win32' ? 'C:\\tmp' : '/tmp';
-
 describe('GithubUrlReader', () => {
   const worker = setupServer();
   setupRequestMockHandlers(worker);
 
-  beforeEach(() => {
-    mockFs({
-      [tmpDir]: mockFs.directory(),
-    });
-  });
-
-  afterEach(() => {
-    mockFs.restore();
-  });
+  beforeEach(mockDir.clear);
```

Comment:
See the comment on a `contents` field on `create` - I think nobody should ever need to write the above line, it should probably instead have default `contents = {}` and resetting itself to that state before every test no matter what. Even fewer footguns, should ideally be "impossible" (for some definition of impossible) to leak state by accident

Reply:
Same here, feeling that's too magic and more likely to cause confusion.

Reply:
But is it the case that if you forget about this line, you get data leakage between tests? Isn't that pretty much unacceptable? 🤔